spec/routing/users_routes_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'UsersController', type: :routing do
+  describe '[GET] #index' do
+    it { expect(get('/')).to route_to('users#index') }
+  end
+
+  describe 'sign in' do
+    it { expect(get: '/users/sign_in').to route_to('devise/sessions#new') }
+  end
+
+  describe 'log out' do
+    it { expect(delete: '/users/sign_out').to route_to('devise/sessions#destroy') }
+  end
```

Comment:
Logging in and logging out are not routes in the user controller,so you can remove it